pyIocApp/setup.c

```diff
@@ -23,6 +23,10 @@
 #include <alarm.h>
 
 #include "pydevsup.h"
+#ifdef _WIN32
+#define PATH_MAX _MAX_PATH
+#include <direct.h>
```

Comment:
Does the order of these two lines change the behavior of `direct.h`?

Reply:
No, I don't think so. It's more appropriate the other way round.
And _MAX_PATH is actually defined in stdlib.h.